getting-started/install.md

```
@@ -0,0 +1,231 @@
+
+## Common issues
+
+:::tip
+Did you have Java installed before, then install the one from this guide? Or uninstall every Java and now you're about to install the one from this guide?
+
+**Restart your PC**. It often saves a lot of debugging.
+:::
+
+### "Temurin 21 is installed, but it says I don't have Java"
+
+You probably skipped the *Custom Setup* screen. Reinstall Java 21 and follow the [Windows steps above](#installing-java-21-recommended) carefully.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)